web/app/scripts/views/record/add-edit-partial.html

```diff
@@ -42,7 +42,7 @@
                             </div>
                         </div>
                         <div class="col-md-12">
-                            <div class="map" leaflet-map driver-embed-map style="height:300px;"></div>
+                            <div class="map" leaflet-map driver-embed-map editable="true"></div>
```

Comment:
I think it might be a slightly better experience for the map to go ahead and load, then animate the pan/zoom to the marker when it comes in, rather than have the page height suddenly expand to show the fully loaded map after other things have rendered.